### ISSUE TITLE
Sidebar tabs appear stacked on top of each other

### ISSUE THREAD

**Issue by arcanistzed** [foundryvtt/foundryvtt#7420]
### Environment Details

* Foundry VTT Version: 10.272, 9.269, and I believe previous versions had the issue
* Operating System: At least Windows 10 & Linux Mint, but most likely every OS
* How Are You Using Foundry: Native Application (Electron), Chrome, and most likely other environnements
* Which Game System: dnd5e and other systems
* Modules Enabled?: both yes and no

### Issue Description

| ![example](https://user-images.githubusercontent.com/82790112/177015807-a43bb36d-d96a-4263-9261-92bdb1732dd1.png) | ![example](https://user-images.githubusercontent.com/82790112/177015874-1fe3e083-224e-4221-ad63-a80141d9a15b.png) |
|-|-|

Two sidebar tabs (or possibly very rarely more than that) can appear stacked on top of each other (both have the `.active` class) until the non-active tab is re-rendered (but re-rendering doesn't always work).

This issue is difficult to reproduce without modules, but it is possible to do so. There are many similar methods that have been used to reproduce this issue and it has cropped in a lot of module issue trackers (e.g. League-of-Foundry-Developers/foundryvtt-devMode/issues/29, arcanistzed/ctg/issues/14) because of how hard it is to reproduce without modules.

One way that I have recently found that works well enough is to attempt to very quickly switch the sidebar tab after loading in. Slowing down your network connection in DevTools can help.
This also seems to do the trick semi-reliably, especially if you have at least one chat message in the ChatLog:
```js
Hooks.once("renderSidebarTab", (app, [html]) => {
    ui.sidebar.activateTab("settings");
});
```
This does the same thing:
```js
Hooks.once("renderCombatTracker", (app, [html]) => {
    ui.sidebar.activateTab("combat");
});
```
It's also possible to synchronously block the render hook to make the issue occur, but this doesn't work as reliably for me:
```js
Hooks.on("renderCombatTracker", () => {
    const startTime = Date.now();
    const pause = 3000; // pause for 3 seconds
    while (Date.now() < startTime + pause);
});
```
Adding some DOM manipulations inside that `while` loop may help reproduce the issue.

**Comment by zeel01** [comment]
I've encountered this a ton, but like Arc it's hard to reproduce.

Originally, I thought it was a module causing it. But as far as I can tell, modules can just make it happen more, but don't cause it outright.

Dev Mode modules is the one that tends to make it happen most easily. But I've experienced it with no mods at all. It seems like things that slow down rendering of the sidebar make it easier to reproduce.
